dfffrontend/auth.js

```diff
@@ -1,13 +1,16 @@
-const apiUrl = "https://faktorprovera.pks.rs";
+const apiUrl = "http://localhost:5000";
 
 const authForm = document.getElementById("auth_data");
 
 function fetchApiKeyAndPrepareHeaders() {
-    const apiKey = authForm.elements["api_key"].value;
+  const apiKey = authForm.elements["api_key"].value;
```

Comment:
Check here if localstorage contains the api_key already. If yes, use that. Otherwise do:
const apiKey = authForm.elements["api_key"].value;
localStorage.setItem("api_key", apiKey);

and then continue